### DISCUSSION TITLE
initial connection failed

### DISCUSSION THREAD

**Discussion by samuel100** [community/community#3582] [Codespaces]
My devcontainer.json is very simple:

{
	"name": "PyTorch",
	"image": "docker.io/pytorch/pytorch",

	// Add the IDs of extensions you want installed when the container is created.
	"extensions": [
		"ms-python.python",
		"ms-python.vscode-pylance"
	]
		
}

However, I get an "initial connection failed [connectionId: blah]. Failed to start VS code  remote server. There is no way to access the logs to see why it is failing.

![image](https://user-images.githubusercontent.com/9161628/111771719-f10add80-88a3-11eb-824c-d1201e1b34d3.png)

This devcontainer.json works fine in VS Code Remote - Containers.

**Comment by 2percentsilk** [comment]
Hi @samuel100,

Is this still occurring? We had an intermittent issue last week that has since resolved. 

If it is still occurring, can you attach the browser console logs?
- Open the Developer Tools window for your browser while on the problem codespaces page (e.g. stuck splash screen).
- Open the Console tab and navigate to Errors.
- Take a screenshot and attach here. Alternatively, you can save the errors as a text file if there are many.
